components/security/authorization.rst

```diff
@@ -49,14 +49,17 @@ recognizes several strategies:
 ``unanimous``
     only grant access if none of the voters has denied access;
 
+``priority``
```

Comment:
We also need a `versionadded` directive

Reply:
You can see how here https://symfony.com/doc/current/contributing/documentation/format.html#new-features-behavior-changes-or-deprecations